template.tpl

```diff
@@ -818,7 +812,6 @@ return {
   anonymousTracking: anonymousTracking,
   contexts: {
     webPage: data.webPage,
-    performanceTiming: data.performanceTiming,
```

Comment:
Likewise, I would kind of expect this just to be updated to `performanceNavigationTiming` rather than be removed outright.

Reply:
Also coming (very) soon.